src/dev/konnector.md

```diff
@@ -2,110 +2,376 @@
 
-But you may need some other npm packages not integrated in [cozy-konnector-libs] to help you run your connector :
+Besides, you'll probably need some other npm packages to help you run your connector:
 
  - [momentjs](http://momentjs.com/docs/) or [date-fns](https://date-fns.org) to manage dates
  - [bluebird](http://bluebirdjs.com) to get enhanced promises
 
-When the connector is started, it also gets some data through environment variables:
+When the connector is started, it also receives some data via environment variables:
 
- - `COZY_CREDENTIALS` : the auth token used by [cozy-client-js] to communicate with the server
- - `COZY_URL` : the API entry point
- - `COZY_FIELDS` : the settings coming from [Cozy Collect] and filled by the user of the connector (login, password, directory path).
+ - `COZY_CREDENTIALS`: an auth token used by [cozy-client-js] to communicate with the server
+ - `COZY_URL`: the Cozy Stack API entry point
+ - `COZY_FIELDS`: settings coming from [Cozy Collect] and filled by the user (login, password, directory path).
 
-Those variables are used by the BaseKonnector and the cozy-client to configure the connection to
-the [Cozy Stack] with the right permissions as defined in your manifest.konnector. These are
-simulated in standalone mode so that you don't need a real cozy stack to develop your connector. [[More information on BaseKonnector](https://github.com/cozy/cozy-konnector-libs/blob/master/packages/cozy-konnector-libs/docs/api.md#basekonnector)]
+These variables are used by the connector and the cozy-client to configure the connection to the [Cozy Stack] with the right permissions as defined in the connector manifest.
+They are simulated in standalone mode so that you do not need a real Cozy Stack to develop a connector.
+[[More about BaseKonnector](https://github.com/cozy/cozy-konnector-libs/blob/master/packages/cozy-konnector-libs/docs/api.md#basekonnector)]
 
-From the server point of view, each connector is a `job` that is run periodically via a `trigger`. [[More information on jobs](https://cozy.github.io/cozy-stack/jobs.html)]
+From the server point of view, each connector is a `job` which is executed periodically via a `trigger`.
+[[More about Cozy Stack jobs](https://cozy.github.io/cozy-stack/jobs.html)]
 
 ## Let’s create our first connector
 
-The easiest way to create a new connector is to use [cozy-konnector-template](https://github.com/cozy/cozy-konnector-template):
+The easiest way to create a new connector is to use [cozy-konnector-template]:
 
-### [cozy-konnector-template] and standalone mode
+### Run the sample
+
+First of all, [download](https://github.com/konnectors/cozy-konnector-template/archive/master.zip) or clone the repository:
 
 ```sh
-git clone https://github.com/cozy/cozy-konnector-template cozy-konnector-monservice
+git clone https://github.com/konnectors/cozy-konnector-template cozy-konnector-newservice
 cd cozy-konnector-monservice
+rm -rf .git
+git init
 yarn # or npm install
```

Comment:
How about putting `yarn install` here? I know they are the same, but i feel like someone who doesn't know yarn will understand `yarn install` more than just `yarn`.

Reply:
Following this advice, we have to write `rm --recursive --force` instead of `rm -rf`, don't you think?